### PR TITLE
fix: fixed reqs for package publish

### PR DESCRIPTION
**Pull request by jhomlala**
Changelog:
* Fixed requirements for package publish

### CODE REVIEW

File: packages/alice_objectbox/pubspec.yaml [hunk 1]
```diff
@@ -11,13 +11,14 @@ environment:
...
     sdk: flutter
-  alice: ^1.0.0-dev.6
+  alice: ^1.0.0-dev.8
   json_annotation: ^4.9.0
   meta: ^1.9.1
   objectbox: ^4.0.1
   objectbox_flutter_libs: ^4.0.1
   path: ^1.9.0
   path_provider: ^2.1.3
+  flat_buffers: ^23.5.26
```
Comment (techouse) [inline packages/alice_objectbox/pubspec.yaml:21]: Not sure why this is need. This is a default transitive dependency and should probably not be directly depended on.

**Review by techouse — CHANGES_REQUESTED** [review]

File: packages/alice_objectbox/pubspec.yaml (hunk 1 above)
Comment (techouse) [reply packages/alice_objectbox/pubspec.yaml:21]: Ah I see, the generated file references it directly
```
Package validation found the following error:
* line 11, column 1 of lib/objectbox.g.dart: This package does not have flat_buffers in the `dependencies` section of `pubspec.yaml`.
     ╷
  11 │ import 'package:flat_buffers/flat_buffers.dart' as fb;
     │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     ╵
```

**Review by techouse — APPROVED** [review]

**Comment by techouse** [comment]
LGTM

File: packages/alice_objectbox/pubspec.yaml (hunk 1 above)
Comment (jhomlala) [reply packages/alice_objectbox/pubspec.yaml:21]: Exactly :)

**Pull request merged**